### PR TITLE
PLAT-750: non signer acdc config

### PR DESCRIPTION
**Pull request by alecsavvy**
### Description
Adds a new config option when deploying the chain `--isminer`. 

When set to true (default) this will unset the false flag if it previously exists in whatever .env file is used. The node will then default to config.cfg and use what's in there on a redeploy.

When set `--ismining=False` the isMining env var is set to false and the node will deploy in non sealing mode.
docs: https://docs.nethermind.io/nethermind/ethereum-client/configuration/init

The effect of this is now nodes that are deployed as non signers will correctly report healthy instead of of attempting to seal blocks and fail for eternity.

Tested on prod ACDC

### CODE REVIEW

**Review by isaacsolo — APPROVED** [review]

**Review by isaacsolo — CHANGES_REQUESTED** [review]
still hesitant about merging this because i think it might add complexity. if our sandbox nodes are the only ones that would want this, it get confusing on knowing whether a node is supposed to be able to mine or not (config.cfg says IsMining true but there happens to be an override env for NETHERMIND_INITCONFIG_ISMINING false. most cases we just want a sandbox to resemble prod. just changing the config.cfg to false seems more obvious? like maybe leave a comment on it that says switch to false if you want to see a healthy chain for non signer.

File: audius-cli [hunk 1]
```diff
@@ -740,6 +742,14 @@ def launch_chain(ctx):
         json.dump(peers, f, ensure_ascii=False, indent=2)
         f.write("\n")
 
+    # override .env with isMining here
+    if not isminer:
+        print("starting node in non mining mode")
+        dotenv.set_key(env, "NETHERMIND_INITCONFIG_ISMINING", "false")
+    else:
+        print("removing ismining env var in favor of config.cfg")
```
Comment (isaacsolo) [inline audius-cli:750]: by default, we will see this log when chain is launched?

unset_key

File: audius-cli [hunk 2]
```diff
@@ -740,6 +742,14 @@ def launch_chain(ctx):
...
+        dotenv.unset_key(env, "NETHERMIND_INITCONFIG_ISMINING")
```
Comment (isaacsolo) [inline audius-cli:751]: what happens if this key doesn't exist?

**Comment by cheran-senthil** [comment]
Please tag me in these PRs that touch audius-cli 👀

Currently, if this bricks, auto upgrade bricks and that is a massive hassle to deal with

**Comment by cheran-senthil** [comment]
This looks, dangerous 👀 you shouldn't be changing audius-cli for this kind of change, I agree much with Isaac here

**Review by cheran-senthil — CHANGES_REQUESTED** [review]
Use set config

**Comment by alecsavvy** [comment]
closing as all nodes should be signers outside of dev / debug machines

**Pull request closed without merging**